Pellicola/Classes/PellicolaCache.swift

```diff
@@ -7,12 +7,14 @@
 
 import Foundation
 
+@objcMembers
 public final class PellicolaCache: NSObject {
     private let fileHandler = PellicolaFileHandler()
+    public var expirationDate = Date()
```

Comment:
I would make this private and add the appropriate init

Reply:
Ok